examples/box/helpers.ts

```diff
@@ -0,0 +1,51 @@
+import * as ContentDisposition from "content-disposition";
+import type { Folder } from "./types";
+import type { User } from "./types";
+import * as coda from "@codahq/packs-sdk";
+import * as mime from "mime-types";
+
+export async function searchFolders(context: coda.ExecutionContext,
+  search: string): Promise<Folder[]> {
+  let url = coda.withQueryParams("https://api.box.com/2.0/search", {
+    type: "folder",
+    query: search || "folder",
```

Comment:
Is searching for "folder" a meaningful thing to do as a default?

Reply:
It's a workaround to search for all folders, since they don't allow an empty query string. Added a comment to make this clearer.